pkg/image-manager/handlers.go

```diff
@@ -370,9 +369,9 @@ func (h *Handlers) deleteImageByName(params image.DeleteImageByNameParams, princ
 	}
 	e.Delete = true
```

Comment:
remove this flag, I think it is duplicated with e.Status

Reply:
see earlier comment.